src/components/FamilyFilter.js

```diff
@@ -350,6 +353,7 @@ class FamilyFilter extends Component {
             </Grid>
           }
         />
+        {this.columns?.confirmationNo === "H" && (
```

Comment:
Is this condition correct? was expecting `!this.columns?.confirmationNo === "H" ...`

Can we also change to this form ? `this.columns?.confirmationNo !== "H"`